<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WhiteBoardDaemonClass</class>
 <widget class="QMainWindow" name="WhiteBoardDaemonClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WhiteBoardDaemon</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="CamView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>531</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <widget class="QMenu" name="menuManual_covering_area">
     <property name="title">
      <string>Manual covering area</string>
     </property>
     <addaction name="actionAdd"/>
     <addaction name="actionErase"/>
    </widget>
    <widget class="QMenu" name="menuManual_Calibration">
     <property name="title">
      <string>Manual Calibration</string>
     </property>
     <addaction name="action25_points"/>
     <addaction name="action36_points"/>
    </widget>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionConfiguration"/>
    <addaction name="actionSpot_Measurement"/>
    <addaction name="actionAutomatic_blocking_noise"/>
    <addaction name="menuManual_covering_area"/>
    <addaction name="menuManual_Calibration"/>
    <addaction name="actionAutomatical_Calibration"/>
    <addaction name="separator"/>
    <addaction name="actionProhibit_light_pen_control"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSet_up">
    <property name="title">
     <string>Setup</string>
    </property>
    <addaction name="actionCamera_Debug_Mode"/>
    <addaction name="actionLaser_Debug_Mode"/>
    <addaction name="actionIn_use_Mode"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionLanguage"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuActions"/>
   <addaction name="menuSet_up"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
  <action name="actionAutomatic_blocking_noise">
   <property name="text">
    <string>Automatical blocking noise</string>
   </property>
  </action>
  <action name="actionAutomatical_Calibration">
   <property name="text">
    <string>Automatical Calibration</string>
   </property>
  </action>
  <action name="actionProhibit_light_pen_control">
   <property name="text">
    <string>Prohibit light-pen control</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionErase">
   <property name="text">
    <string>Erase</string>
   </property>
  </action>
  <action name="action25_points">
   <property name="text">
    <string>25 points</string>
   </property>
  </action>
  <action name="action36_points">
   <property name="text">
    <string>36 points</string>
   </property>
  </action>
  <action name="actionCamera_Debug_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Camera Debug Mode</string>
   </property>
  </action>
  <action name="actionLaser_Debug_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Laser Debug Mode</string>
   </property>
  </action>
  <action name="actionIn_use_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Normal Service Mode</string>
   </property>
  </action>
  <action name="actionLanguage">
   <property name="text">
    <string>Language</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSpot_Measurement">
   <property name="text">
    <string>Spot Measurement</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="whiteboarddaemon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionConfiguration</sender>
   <signal>triggered()</signal>
   <receiver>WhiteBoardDaemonClass</receiver>
   <slot>configure()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAutomatical_Calibration</sender>
   <signal>triggered()</signal>
   <receiver>WhiteBoardDaemonClass</receiver>
   <slot>autoCalibration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>WhiteBoardDaemonClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
